### DISCUSSION TITLE
DataTable empty cells are undefined

### DISCUSSION THREAD

**Discussion by annette-graham** [primefaces/community#2496] [PrimeReact] [answered]
Hi, I was just wondering if there is anything inbuilt in the DataTable component that will handle when rows have empty fields?

At the moment, if a user wants to add a new row to my table and they don't fill a value for each column it dispalys "undefined", but I would like the cell to just be empty. 

Thanks!

**Comment by melloware** [comment, marked as answer]
Can't you just handle that with `bodyTemplate=` on your Column and just render whatever you want if its undefined?

**Reply by annette-graham** [reply]
Yep most probably! My bad. One of those days apparently 😅
